README.md

```diff
@@ -217,6 +217,23 @@ All requests average: 524ms
 
 If `OUTPUT_DATE` is not provided, then it will output a summary for all results within the provided directory.
 
+### Summarise Stress Test results:
```

Comment:
The previous heading should be updated to differentiate it from the Stress Test results. Something like `Summarise the Daily Test results`